packages/eslint-plugin/tests/lib/config/all.test.js

```diff
@@ -4,7 +4,9 @@ describe('config', () => {
   it('has valid plugins and requires', () => {
     expect(
       execESLint(
-        `--config ${fixtureFile('all/.eslintrc.js')} ${fixtureFile('all')}`,
+        `--ext .js,.graphql --ignore-pattern "**/all/build/*" --config ${fixtureFile(
+          'all/.eslintrc.js',
+        )} ${fixtureFile('all')}`,
       ),
     ).toBe('');
   }, 8000);
```

Comment:
do we worry this timeout mike make our tests flaky at some point?

Reply:
I didn't add it but I assume it was expanded to stop it from being flakey in CI environments.